index.bs

```diff
@@ -216,6 +217,53 @@ If the Client does not use an identifier that can be dereferenced, then it MUST
 registered with the OP via either OIDC dynamic or static registration.
 See also [[!OIDC.DynamicClientRegistration]].
 
+# WebID Document # {#webid-document}
+
+Dereferencing WebID results in WebID Document.
+
+Issue(76):
+
+## OIDC Issuer Discovery ## {#oidc-issuer-discovery}
+
+To prevents a malicious OpenID Provider from issuing valid OIDC ID Tokens for arbitrary WebIDs.
```

Comment:
```suggestion
A WebID Profile lists the OpenID Providers who are trusted to issue tokens on behalf
of the agent who controls the WebID. This prevents a malicious OpenID Provider from issuing
otherwise valid ID Tokens for arbitrary WebIDs. An entity that verifies ID Tokens will use this
mechanism to determine if the issuer is authoritative for the given WebID.
```